jwst/extract_1d/extract.py

```diff
@@ -1180,18 +1229,18 @@ def offset_from_offset(self, input_model, slit, verbose):
             The pixel coordinate of the target in the cross-dispersion
             direction, at the middle of the spectrum in the dispersion
             direction.
-
-        verbose : bool
-            If True, log messages.
         """
 
+        targ_ra, targ_dec = self.get_target_coordinates(input_model, slit)
+        if targ_ra is None or targ_dec is None:
+            return (0., None)
+
         # Use the WCS function to find the cross-dispersion (XD) location that
         # is closest to the target coordinates.  This is the "actual" location
         # of the spectrum, so the extraction region should be centered here.
-        (middle, middle_wl, locn) = self.locn_from_wcs(
-                                        input_model,
-                                        slit,
-                                        verbose)
+        locn_info = self.locn_from_wcs(input_model, slit, targ_ra, targ_dec,
+                                       verbose)
+        (middle, middle_wl, locn) = locn_info
```

Comment:
```suggestion
        middle, middle_wl, locn = locn_info
```
Don't need the parentheses here